src/DynamoCore/Search/Legacy/CategoryBuilder.cs

```diff
@@ -0,0 +1,420 @@
+﻿using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.Linq;
+using Dynamo.Search.SearchElements;
+using Dynamo.UI;
+
+namespace Dynamo.Search
+{
+#if false
```

Comment:
If this file is not needed, why don't we remove it now? Also its entry in `*.csproj`, if there's any.